### ISSUE TITLE
[BUG] Android API not working for Huawei P40 lite

### ISSUE THREAD

**Issue by OxNinja** [microsoft/playwright#8249]
**Context:**
- Playwright Version: `1.12.2-1623435892000`
- Operating System: Linux
- Node.js version: `12.22.5`
- Browser: All
- Extra:

**Code Snippet**

Help us help you! Put down a short code snippet that illustrates your bug and
that we can run and debug locally. For example:

```javascript
const {_android: android} = require('playwright');

(async () => {
  const [device] = await android.devices(); // Got my Huawei P40 lite connected with ADB
  await device.launchBrowser(); // Nothing on the device
})();
```

**Describe the bug**

The `AndroidDevice.launchBrowser()` method does not work on this device in particular. It is brand new if this can help.

**Comment by JoelEinbinder** [comment]
Does it work for you with other devices or is the P40 the only one you have trouble with?

Can you verify that you did each step from the [docs](https://playwright.dev/docs/api/class-android):

- ADB daemon running and authenticated with your device. Typically running `adb devices` is all you need to do.
- Chrome 87 or newer installed on the device
- "Enable command line on non-rooted devices" enabled in chrome://flags.

**Comment by OxNinja** [comment]
Yes it works fine with other devices. Tested with Motorola g9, Samsung J6 and Tab A6.

I will try to work with `device.launchBrowser({command: "com.example.navigator"})` as Chrome is not installed by default on Huawei.

**Comment by JoelEinbinder** [comment]
Chrome not being installed sounds like the issue. You can try with `command: "com.huawei.browser"` but I can't guarantee it will work. We should probably have a nicer error if Chrome is not installed.

**Comment by OxNinja** [comment]
Yes I tried with both `com.huawei.browser` and `com.huawei.browserhomepage` with no result. Do you plan supporting other browser than Chrome?
I will install it on the device.

**Comment by OxNinja** [comment]
Update: as for the moment I don't have Google services on the device I tried with Firefox. The following do not work, like for `com.huawei.browser`:
```js
await device.launchBrowser({command: "org.mozilla.firefox"});
```
I presume that Playwright is meant to work only with Chrome? I will dig in the source code again if I did something wrong

**Comment by OxNinja** [comment]
To ensure that I understand correctly Playwright:

When calling `AndroidDevice.launchBrowser()` which is defined here https://github.com/microsoft/playwright/blob/c4eb2d4b1e17a0de1cea7220be45545c7c524614/src/dispatchers/androidDispatcher.ts#L154-L157

I guess that the `command` attribute is wrong and we should use `pkg` instead. Then this is called https://github.com/microsoft/playwright/blob/c4eb2d4b1e17a0de1cea7220be45545c7c524614/src/client/android.ts#L229-L235

Where the parameters are prepared with https://github.com/microsoft/playwright/blob/c4eb2d4b1e17a0de1cea7220be45545c7c524614/src/client/browserContext.ts#L355-L374

To be sure, `pkg` is a String https://github.com/microsoft/playwright/blob/827a909d36b7e5735d411ad5a4f7e1c91395eb94/src/protocol/channels.ts#L3159-L3160

But when using the following code:

```js
// in async function
await device.launchBrowser({ pkg: "org.mozilla.firefox" })
```

Nothing happens on the device. Firefox is installed, I also checked with `pm list packages | grep firefox`.
